server/src/services/tooljet_db.service.ts

```diff
@@ -163,12 +165,14 @@ export class TooljetDbService {
       let query = `${column['column_name']} ${column['data_type']}`;
       if (column['column_default']) query += ` DEFAULT ${this.addQuotesIfString(column['column_default'])}`;
       if (column['constraint_type']) query += ` ${column['constraint_type']}`;
+      if (column['isNotNull']) query += ` NOT NULL`;
```

Comment:
Does it make sense to change the request body so there can be multiple constraints for creating a table?
We can reuse the constraint_type key's value to accommodate this. It will adhere to SQL syntax itself ie. primary key, not null, uniqueness are all constraints that can be added for a column at hand.